examples/v1/firmware/firmware.go

```diff
@@ -18,13 +17,13 @@ func main() {
 	defer cancel()
 
 	// set BMC parameters here
-	host := ""
```

Comment:
Want to copy/paste the `flag` imports from the other examples?

Reply:
will include the `flag` imports at rebase in the parent branch.